modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -1777,7 +1783,7 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
     const response: OfflineVaultTxInfo = {
       txHex: tx.toBroadcastFormat(),
       userKey,
-      coin: token,
+      coin: token ? token : this.getChain(),
```

Comment:
please test this change for hot and cold wallets before merging

Reply:
tested for hot wallet (supported and unsupported token).
will test for cold as well.

Reply:
tested for cold wallet unsupported token. it's working as expected.